src/com/tony/server/Main.java

```diff
@@ -38,12 +38,15 @@ public static void main(String[] args) {
                 uriList.add("/" + listOfFiles[i].getName());
             }
         }
+        uriList.remove("/.DS_Store");
+        uriList.remove("/form");
+        uriList.remove("/method_options");
         return uriList;
```

Comment:
Can you pull this out into a config file of some sort? Our customer shouldn't have to recompile/redeploy just to ignore some files.